src/sidebar/util/copy-to-clipboard.js

```diff
@@ -10,10 +10,13 @@
  * @param {string} text
  */
 export function copyText(text) {
-  const temp = document.createElement('pre');
-  temp.className = 'copy-text';
-  temp.textContent = text;
+  const temp = document.createElement('input');
+  temp.value = text;
+  temp.setAttribute('data-test-id', 'copy-text');
```

Comment:
```suggestion
  temp.setAttribute('data-test-id', 'copy-text');  // used as a selector for testing
```

Reply:
@robertknight [suggested to us `data-testid`](https://github.com/hypothesis/lms/pull/2146#discussion_r509171217) at it seems it is the standard attribute for testing purposes.